cryostatio/cryostat-web
# feat(rules): allow cleanup resources when disabling automated rules

# Welcome to Cryostat! 👋
## Before contributing, make sure you have:
_______________________________________________

Related to https://github.com/cryostatio/cryostat/pull/1289

## Description of the change:

- [x] Added warning modal for users to decide whether to clean up resources created by the automated rule.
- [x] Extend delete warning modals to supporting disabling.
- [x] Fix side-effects on mockRule in tests.

## Motivation for the change:

See https://github.com/cryostatio/cryostat/issues/1283

## How to manually test:
1. Go to Automated Rule tab
2. Create or enable a existing rule.
3. Now toggle the rule to disable it. The modal should show up.
